### ISSUE TITLE
[BBB-2.3.10 and BBB 2.4-beta-2] Breakout room "allow users to choose a breakout room to join" not working

### ISSUE THREAD

**Issue by 1stbutch** [bigbluebutton/bigbluebutton#12966]
**Describe the bug**
There seems to be a problem with breakout room creation and letting users choose the room they want to join.
If you select "Allow users to choose a breakout room to join", they only get a deep blue site, but not the interface to choose the room. They also need to rejoin the current session. (see screenshot below)
If you assign users to specific breakout rooms, everything works fine.

**To Reproduce**
1. Go to 'demo.bigbluebutton.org'
2. Join with at least one additional non moderator user
3. Create breakout rooms and select "Allow users to choose a breakout room to join"
4. Non moderator user should be redirected to "Choose a breakout room to join" site but will only see a deep blue screen. (screenshot below)

**Expected behavior**
Users should see a drop-down menu where they can select a breakout room to join

**Actual behavior**
Users only see a deep blue site and need to rejoin the current conference.

**Screenshots**
![error-bbb-grouproom_2021-08-11 070800](https://user-images.githubusercontent.com/14950232/128973408-912c3b44-d7f9-40b8-8f5b-bf8a9c288567.png)

**Desktop (please complete the following information):**
 - OS: [Windows (21H1) and Linux (Ubuntu 20.04)]
 - Browser [tested with Firefox, Chrome and Microsoft Chromium Edge]
 - Version [Firefox: 91.0, Chrome: 92.0.4515.131. Microsoft: 92.0.902.67]

**Smartphone (please complete the following information):**
 - Device: [Google Pixel 4a]
 - OS: [Android 11]
 - Browser [Firefox]
 - Version [91.1.0]

**Additional context**
We have several BBB servers running. Those with BBB version 2.3.8 seem to work fine.
I think the problem first appeared with version 2.3.9.

**Comment by ffdixon** [comment]
Thanks for reporting - we're able to reproduce this on demo.bigbluebutton.org.  The upcoming 2.3.11 (which will be released soon) does not have this bug and we'll be releasing it soon.

**Comment by ffdixon** [comment]
Update: We're running 2.4-beta-2 on our demo servers, and that bug will also be fixed in the upcoming 2.4-beta-3.

**Comment by ffdixon** [comment]
Can you upgrade to beta-3 and let us know if this is still an issue.

**Comment by 1stbutch** [comment]
Thanks for the fast response.

I just upgraded our testing system to 2.4-beta-3.

If I create a breakout room and let users choose which room to enter, i get the following screen:

![grouproom_error01](https://user-images.githubusercontent.com/14950232/129443482-4c2e2d49-c0f4-4a6c-b929-b30e48c8d9c0.jpg)

If I just click "Join Room" and don't change the selected room, I end up in a new browser tab with "about:blank" as url.
If I select "Room 2", I get the following situation and joining works without any problems.

![grouproom_error02](https://user-images.githubusercontent.com/14950232/129443519-181f001d-5523-4e29-8119-fd4ce37f7af1.jpg)

If I change from "Room 1" to "Room 2" or whichever is selected first, it looks like the system generates the correct room url and joining the breakout room works.

I also noticed that joining with a mobile device (only got an android 11 device) doesn't seem to work at all. I can select a breakout room and for less than a second i can see the "... connecting" field before it is gone and only the deep blue background is visible. (no chat, no presentation)

I also checked our BBB-2-3 testing system and updated to 2.3.12.
Creating and joining breakout rooms works without any problems. (also on mobile device)
The system also directly generates the breakout room url when the selection screen comes up. (without the need to change the first selected room)
